mora/service/org.py

```diff
@@ -761,3 +765,77 @@ def get_org_unit_history(unitid):
                                unit_registrations))
 
     return flask.jsonify(history_entries)
+
+
+@blueprint.route('/o/<uuid:orgid>/address_autocomplete/')
```

Comment:
Minor nit: I was a bit surprised to see this in the `org` module — doesn't it belong to `address`?

(Also, having it here triggers a merge conflict due to my recent changes.)

Reply:
I moved it to `org` because of the change in endpoint address - I'm not sure how that made sense to me. I'll change it back.